deploy/frontend.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: patchman-ui
+objects:
+  - apiVersion: cloud.redhat.com/v1alpha1
+    kind: Frontend
+    metadata:
+      name: patch
+    spec:
+      envName: ${ENV_NAME}
+      title: Patch
+      deploymentRepo: https://github.com/RedHatInsights/patchman-ui
+      API:
+        versions:
+          - v1
+      frontend:
+        paths:
+          - /apps/patch
+      image: ${IMAGE}:${IMAGE_TAG}
+      navItems:
+        - appId: "patch"
+          title: "Patch"
+          href: "/insights/patch"
+          product: "Red Hat Insights"
+          expandable: true
+          routes:
+            - appId: 'patch'
+              title: 'Advisories'
+              href: '/insights/patch/advisories'
+            - appId: 'patch'
+              title: 'Systems'
+              href: '/insights/patch/systems'
+            - appId: 'patch'
+              title: 'Packages'
+              href: '/insights/patch/packages'
+            - appId: 'patch'
+              title: 'Patch Template'
```

Comment:
Should this be template or patch-set? https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/rhel-navigation.json#L72 . I want to make sure I'm looking at the right source file for the nav.

Reply:
Initially we were using Patch Set, but recently UX requested to rename to 'Patch template'. I have a ready PR to rename all titles on Patch. I wanted to rename the nav also directly with this PR.